src/pykiso/test_result/multi_result.py

```diff
@@ -0,0 +1,239 @@
+
+        :param test: running testcase
+        """
+        for result in self.result_classes:
+            result.addUnexpectedSuccess(test)
+
+    def addExpectedFailure(
+        self,
+        test: Union[BasicTest, BaseTestSuite],
+        err: ExcInfoType,
+    ) -> None:
+        """Call the addExpectedFailure function for all result classes.
+
+        :param test: running testcase
+        :param err: tuple returned by sys.exc_info
+        """
+        for result in self.result_classes:
+            result.addExpectedFailure(test, err)
+
+    def addSubTest(
+        self,
+        test: Union[BasicTest, BaseTestSuite],
+        subtest: _SubTest,
+        err: ExcInfoType,
+    ) -> None:
+        """Call the addSubTest function for all result classes.
+
+        :param test: running testcase
+        :param subtest: subtest runned
+        :param err: tuple returned by sys.exc_info
+        """
+        for result in self.result_classes:
+            result.addSubTest(test, subtest, err)
+
+    def addError(
+        self,
+        test: Union[BasicTest, BaseTestSuite],
+        err: ExcInfoType,
+    ) -> None:
+        """Call the addError function for all result classes.
+
+        :param test: running testcase
+        :param err: tuple returned by sys.exc_info
+        """
+        for result in self.result_classes:
+            result.addError(test, err)
+
+    def addFailure(
+        self,
+        test: Union[BasicTest, BaseTestSuite],
+        err: ExcInfoType,
+    ) -> None:
+        """Call the addFailure function for all result classes.
+
+        :param test: running testcase
+        :param err: tuple returned by sys.exc_info
+        """
+        for result in self.result_classes:
+            result.addFailure(test, err)
+
+    def printErrorList(self, flavour: str, errors: List[tuple]) -> None:
+        """Call printErrorList function once, it will first try to call the
+        function for a BannerTestResult first but if none are defined it will
+        call the function of the first result class.
+
+        :param flavour: failure reason
+        :param errors: list of failed tests with their error message
+        """
+        for result in self.result_classes:
+            if isinstance(result, BannerTestResult):
+                return result.printErrorList(flavour, errors)
+        return self.result_classes[0].printErrorList(flavour, errors)
+
+    def printErrors(self) -> None:
+        """Call printErrors function once, it will first try to call the function
+        for a BannerTestResult first but if none are defined it will call the
+        function of the first result class.
+        """
+        for result in self.result_classes:
+            if isinstance(result, BannerTestResult):
+                return result.printErrors()
+        return self.result_classes[0].printErrors()
+
+    def getDescription(
+        self,
+        test: Union[BasicTest, BaseTestSuite],
```

Comment:
does black do this? it's looks like that the total characters doesn't exceed black rule

Reply:
It was done by black because I had a comma at the end.